src/cart/crt_bulk.c

```diff
@@ -237,5 +265,5 @@ crt_bulk_access(crt_bulk_t bulk_hdl, d_sg_list_t *sgl)
 int
 crt_bulk_abort(crt_context_t crt_ctx, crt_bulk_opid_t opid)
 {
-	return crt_hg_bulk_cancel(opid);
+	return HG_Bulk_cancel(opid);
```

Comment:
shouldn't you do the conversion here too ?

Reply:
I did wonder about this.  It looks like this function is only called from one location and that does not check it's return code.